_security/access-control/users-roles.md

```diff
@@ -27,30 +29,13 @@ Roles are the core way of controlling access to your cluster. Roles contain any
 
 ---
 
-## Creating and editing OpenSearch roles
-
-You can update OpenSearch by using one of the following methods.
-
-### Using the API
-
-You can send HTTP requests to OpenSearch-provided endpoints to update security roles, permissions, and associated settings. This method offers granular control and automation capabilities for managing roles.
-
-### Using the UI (OpenSearch Dashboards)
+## Defining users
 
-OpenSearch Dashboards provides a user-friendly interface for managing roles. Roles, permissions, and document-level security settings are configured in the Security section within OpenSearch Dashboards. When updating roles through the UI, OpenSearch Dashboards calls the API in the background to implement the changes.
+You can define users using OpenSearch Dashboards, `internal_users.yml`, or the REST API. When creating a user, you can map users to roles using `internal_users.yml` or the REST API, but that feature is not currently available in OpenSearch Dashboards.
```

Comment:
Is it true that you the mapping feature is not available in Dashboards? Because later we provide instructions on it.